duck/__init__.py

```diff
@@ -19,4 +21,6 @@
     'Spy',
     'Stub',
     'stub',
+    'ANY',
```

Comment:
Not going to block on it, but in general I prefer keep these in alphabetical order.